### PR TITLE
Fix 'Getting Started with Rails' [ci skip]

### PR DESCRIPTION
**Pull request by bogdanvlviv**
`bin/rails generate controller Comments` creates files
app/assets/javascripts/comments.coffee
app/assets/stylesheets/comments.scss
not
app/assets/javascripts/comment.coffee
app/assets/stylesheets/comment.scss

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -1655,8 +1655,8 @@ This creates five files and one empty directory:
 | app/views/comments/                          | Views of the controller are stored here  |
 | test/controllers/comments_controller_test.rb | The test for the controller              |
 | app/helpers/comments_helper.rb               | A view helper file                       |
-| app/assets/javascripts/comment.coffee        | CoffeeScript for the controller          |
-| app/assets/stylesheets/comment.scss          | Cascading style sheet for the controller |
+| app/assets/javascripts/comments.coffee        | CoffeeScript for the controller          |
+| app/assets/stylesheets/comments.scss          | Cascading style sheet for the controller |
```
Comment (kaspth) [inline guides/source/getting_started.md]: Please realign the indentation in the second column again 🤓

Comment (bogdanvlviv) [inline guides/source/getting_started.md]: Oops! Thanks!

File: guides/source/getting_started.md [hunk 2]
```diff
@@ -1649,14 +1649,14 @@ $ bin/rails generate controller Comments
...
-| app/assets/javascripts/comment.coffee        | CoffeeScript for the controller          |
-| app/assets/stylesheets/comment.scss          | Cascading style sheet for the controller |
+| File/Directory                                | Purpose                                  |
+| --------------------------------------------  | ---------------------------------------- |
+| app/controllers/comments_controller.rb        | The Comments controller                  |
+| app/views/comments/                           | Views of the controller are stored here  |
+| test/controllers/comments_controller_test.rb  | The test for the controller              |
+| app/helpers/comments_helper.rb                | A view helper file                       |
+| app/assets/javascripts/comments.coffee        | CoffeeScript for the controller          |
+| app/assets/stylesheets/comments.scss          | Cascading style sheet for the controller |
```
Comment (kaspth) [inline guides/source/getting_started.md]: Still not correct in indentation. Delete the extra spaces you added and then remove one space each from the bottom two rows that you added a character to ;)

Comment (bogdanvlviv) [inline guides/source/getting_started.md]: :flushed:

**Comment by kaspth** [comment]
There we go!

**Comment by bogdanvlviv** [comment]
Thanks! :)

**Pull request merged**